bundles/org.openhab.binding.avmfritz/src/main/resources/ESH-INF/i18n/avmfritz.properties

```
@@ -0,0 +1,10 @@
+# thing actions
+setBoostModeModeActionLabel = Set Boost Mode
+setBoostModeActionDescription = Activates the Boost mode of the heating thermostat.
+setBoostModeDurationInputLabel = Duration
+setBoostModeDurationInputDescription = Duration in seconds, min. 1, max. 86400, 0 for deactivation.
+
+setWindowOpenModeActionLabel = Set Window Open Mode
+setWindowOpenModeActionDescription = Activates the Window Open mode of the heating thermostat.
```

```suggestion
setWindowOpenModeActionDescription = Activates the Window Open Mode of the heating thermostat.
```

Should `mode` be capitalized here?

---

I would say no.